_posts/2016-12-19-2016-year-in-review.md

```diff
@@ -0,0 +1,172 @@
+in <a href="http://flink.apache.org/news/2015/12/18/a-year-in-review.html" target="_blank">2015’s year in review post</a>. Here's to making that an annual tradition :)
+
+## Looking ahead to 2017
+
+A good source of information about the Flink community's roadmap is the list of
+<a href="https://cwiki.apache.org/confluence/display/FLINK/Flink+Improvement+Proposals" target="_blank">Flink
+Improvement Proposals (FLIPs)</a> in the project wiki. Below, we'll highlight a selection of FLIPs
+that have been accepted by the community as well as some that are still under discussion.
+
+We should note that work is already underway on a number of these features, and some will even be included in Flink 1.2 at the beginning of 2017.
+
+* **A new Flink deployment and process model**, as described in <a href="https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=65147077" target="_blank">FLIP-6<a/>. This work ensures that Flink supports a wide
+range of deployment types and cluster managers, making it possible to run Flink smoothly in any environment.
+
+* **Dynamic scaling** for both key-value state <a href="https://github.com/apache/flink/pull/2440" target="_blank">(as described in
+this PR)<a/> *and* non-partitioned state <a href="https://cwiki.apache.org/confluence/display/FLINK/FLIP-8%3A+Rescalable+Non-Partitioned+State" target="_blank">(as described in FLIP-8)<a/>, ensuring that it's always possible to split or merge state when scaling up or down, respectively.
+
+* **Asynchronous I/O**, as described in <a href="https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=65870673" target="_blank">FLIP-12
+</a>, which makes I/O access a less time-consuming process without adding complexity or the need for extra checkpoint coordination.
+
+* **Enhancements to the window evictor**, as described in <a href="https://cwiki.apache.org/confluence/display/FLINK/FLIP-4+%3A+Enhance+Window+Evictor" target="_blank">FLIP-4</a>,
+to provide users with more control over how elements are evicted from a window.
+
+* **Fined-grained recovery from task failures**, as described in <a href="https://cwiki.apache.org/confluence/display/FLINK/FLIP-1+%3A+Fine+Grained+Recovery+from+Task+Failures" target="_blank">FLIP-1</a>,
+to make it possible to restart only what needs to be restarted during recovery, building on cached intermediate results.
+
+* **Unified checkpoints and savepoints**, as described in <a href="https://cwiki.apache.org/confluence/display/FLINK/FLIP-10%3A+Unify+Checkpoints+and+Savepoints" target="_blank">FLIP-10</a>, to
+allow savepoints to be triggered automatically--important for program updates for the sake of error handling because savepoints allow the user to modify both
+ the job and Flink version whereas checkpoints can only be recovered with the same job.
+
+* **Table API stream aggregations**, as described in <a href="https://cwiki.apache.org/confluence/display/FLINK/FLIP-11%3A+Table+API+Stream+Aggregations" target="_blank">FLIP-11</a>, to
```

Comment:
I would rephrase to "Table API window aggregations, as described in FLIP-11 to support group-window and row-window aggregates on streaming and batch tables.